AppCenter/AppCenterTests/MSAuthTokenContextTests.m

```diff
@@ -45,4 +45,18 @@ - (void)testSetAuthToken {
   OCMVerify([delegateMock authTokenContext:self.sut didReceiveAuthToken:expectedAuthToken]);
 }
 
+- (void)testRemoveDelegate {
+
+  // If
+  id<MSAuthTokenContextDelegate> delegateMock = OCMProtocolMock(@protocol(MSAuthTokenContextDelegate));
+  [self.sut addDelegate:delegateMock];
+
+  // Then
+  OCMReject([delegateMock authTokenContext:self.sut didReceiveAuthToken:OCMOCK_ANY]);
+
+  // When
+  [self.sut removeDelegate:delegateMock];
+  self.sut.authToken = @"something";
+}
```

Comment:
it's not obvious to me why we are setting the auth token after removing a delegate. I also don't see an actual test after `// When` or maybe I'm missing something?

Reply:
Gosh, I just realized how the test works! Sorry about my comment.